sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/JsonElementJsonConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    internal class JsonElementJsonConverter : JsonConverter<JsonElement>
```

Comment:
Nit - it's kind of hard to read this file because both [Newtonsoft](https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_JsonConverter_1.htm) and [System.Text.Json](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.serialization.jsonconverter-1?view=net-6.0) have classes called `JsonConverter<T>`.  It might be helpful to future readers to either fully qualify like `Newtonsoft.Json.JsonConverter<Sytstem.Text.Json.JsonElement>` or [alias](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive) at the top of the file with `using NewtonsoftConverter = Newtonsoft.Json.JsonConverter<System.Text.Json.JsonElement>`.